src/quartic/pipeline/runner/cli.py

```diff
@@ -32,15 +32,15 @@ def parse_args(argv):
 def run_user_code(f, exception_file):
     try:
         return f()
-    except ModuleNotFoundError as e:
-        raise ModuleNotFoundException(e.name)
+    except ModuleNotFoundException as e:
+        raise ModuleNotFoundException(e.module)
```

Comment:
Is this weird exception conversion wrong?

Reply:
I've reverted to what it was before. If we see again let's fix.